draft-gpew-priv-ppm.md

```diff
@@ -805,10 +802,10 @@ carries is up to the helper implementation.
 
 ### Aggregate Share Request {#aggregate-share-request}
 
-Once the aggregators have verified at least as many reports as required for the
-PPM task, the leader issues an "aggregate-share request" to each helper. The
-helper responds to this request by extracting its aggregate share from its state
-and encrypting it under the collector's HPKE public key.
+Once it has verified at least as many reports as required for the PPM task, the
+leader may issue an "aggregate-share request" to each helper. The helper
+responds to this request by extracting its aggregate share from its state and
```

Comment:
nit: "extracting its aggregate share from its state" implies the helper will have an aggregate share ready to go in its state, which won't be true for `poplar1`. We could just say "The helper responds to this request with its aggregate share, encrypted under the collector's HPKE public key."

Reply:
Actually at this point it will have computed an aggregate share. However I took your suggestion because your text is a lot clearer.